CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### 🐞 Fixed
 - Make SendButton animation overridable [#1781](https://github.com/GetStream/stream-chat-swift/issues/1781)
 - Make ChannelId.rawValue public [#1780](https://github.com/GetStream/stream-chat-swift/pull/1780)
+- Fix channel not removed from channel list when user leaves the channel [#1785](https://github.com/GetStream/stream-chat-swift/pull/1785)
```

Comment:
This is not a fix, but a change in behavior: now default behavior is "ChannelListVC keeps track of channels where user is a member" instead of "all channels loaded in the SDK"

Reply:
Hum 🤔 But in one way, this is going to be easier to find and understand if someone is looking for this issue. I'm going to add that to the change section, but it is worth to leave this one on the fix section as well

Reply:
please keep it as a fix, the previous implementation did not make any sense

Reply:
@tbarbugli  Yes, we leaving it as a fix  👍